instrumentation/cassandra-3.0/src/main/java/io/opentelemetry/auto/instrumentation/datastax/cassandra/TracingSession.java

```diff
@@ -260,7 +260,7 @@ public void run() {
   }
 
   private Span startSpan(final String query) {
-    final Span span = TRACER.spanBuilder("cassandra.query").setSpanKind(CLIENT).startSpan();
+    final Span span = TRACER.spanBuilder(query).setSpanKind(CLIENT).startSpan();
```

Comment:
does this possibly leak PII into the span names?

Reply:
Yeah, similar to our jdbc capture, if you embed PII directly into your query and don't use parameters, then it will end up in span name

Reply:
I see.  Are we ok with this? Does this open the project up to any legal issues if PII leaks because of this decision?

Reply:
Are you asking me a legal question?

Reply:
ha! I'm more wondering if this decision has been discussed with CNCF legal sorts of people, and if that's appropriate.

Reply:
Pretty sure Apache License covers open source projects from liability. Otherwise how does anything work?

Reply:
wait! things work??!!??

Reply:
Btw, we do have an issue already tracking the PII concern, see #193, does that make you comfortable approving this?